### PR TITLE
feat: Expose installed NS mod directory

### PR DESCRIPTION
**Pull request by GeckoEidechse**
This allows other functions to get a mod directory directly which is useful for e.g. deleting a mod.

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
f14ea9b82fd57c355b6957a8280f8025ee85658b additionally makes it easier to add more fields to return in the future.

File: src-tauri/src/mod_management/mod.rs [hunk 1]
```diff
@@ -167,8 +165,17 @@ fn parse_installed_mods(
                 Ok(thunderstore_mod_string) => Some(thunderstore_mod_string),
                 Err(_err) => None,
             };
+        // Get directory path
+        let mod_directory = directory.to_str().unwrap().to_string();
+
+        let ns_mod = NorthstarMod {
+            name: mod_name,
+            thunderstore_mod_string: thunderstore_mod_string,
+            enabled: false, // Placeholder
```
Comment (Alystrasz) [inline src-tauri/src/mod_management/mod.rs:174]: Since Northstar considers mods as enabled by default, shouldn't this be true?

Comment (GeckoEidechse) [reply src-tauri/src/mod_management/mod.rs:174]: Shouldn't matter cause this function is only ever called by [`get_installed_mods_and_properties`](https://github.com/R2NorthstarTools/FlightCore/blob/f14ea9b82fd57c355b6957a8280f8025ee85658b/src-tauri/src/mod_management/mod.rs#L188) where the value is overwritten. ^^

Also I'm planning to combine the two functions (`get_installed_mods_and_properties` and `parse_installed_mods`) into a single one anyway after this is (+ #110 and #111) merged.

**Review by Alystrasz — APPROVED** [review]
Confirmed working on Linux.

**Pull request merged**